Userland/Applications/HexEditor/HexEditor.h

```diff
@@ -116,10 +116,23 @@ class HexEditor : public GUI::AbstractScrollableWidget {
     size_t total_rows() const { return ceil_div(m_content_length, bytes_per_row()); }
     size_t line_height() const { return font().pixel_size_rounded_up() + m_line_spacing; }
     size_t character_width() const { return font().glyph_fixed_width(); }
-    size_t cell_width() const { return character_width() * 3; }
+    size_t cell_gap() const { return character_width() / 2; }
+    size_t cell_width() const { return character_width() * 2 + cell_gap(); }
+    size_t group_gap() const { return character_width() * 1.5; }
+    size_t group_width() const
+    {
+        return (character_width() * 2 * bytes_per_group())
```

Comment:
these are getting close to the "hm, do this out of line" limit for how much is going on.

Reply:
Yeah if they get any more complicated I'll move them.